pom.xml

```diff
@@ -122,6 +122,7 @@
         <!-- alpha order require -->
         <dep.accumulo.version>1.9.0</dep.accumulo.version>
         <dep.bson.version>3.6.3</dep.bson.version>
+        <dep.commons-cli.version>1.4</dep.commons-cli.version>
```

Comment:
Flink uses Commons CLI 1.3 which is incompatible with this version. You can either use the version Flink uses or add a relocation rule to the Shade Plugin.
Something like:
```xml
<relocations>
	<relocation>
		<pattern>org.apache.commons.cli</pattern>
		<shadedPattern>org.gradoop.examples.thirdparty.org.apache.commons.cli</shadedPattern>
	</relocation>
</relocations>
```

Reply:
added relocation rule